src/main/bash/sdkman-main.sh

```diff
@@ -144,27 +141,17 @@ function sdk() {
 	# Native commands found under libexec
 	local native_command="${SDKMAN_DIR}/libexec/${COMMAND}"
 	
-	# Internal commands use underscores rather than hyphens
-	local converted_command_name=$(echo "$COMMAND" | tr '-' '_')
-
 	if [ -f "$native_command" ]; then
-		# Available as native command
-		if [ -z "$QUALIFIER" ]; then
-			"$native_command"
-		elif [ -z "$3" ]; then
-			"$native_command" "$QUALIFIER"
-		elif [ -z "$4" ]; then
-			"$native_command" "$QUALIFIER" "$3"
-		else
-			"$native_command" "$QUALIFIER" "$3" "$4"
-		fi
-		final_rc=$?
+		"$native_command" "${@:2}"
 
 	elif [ -n "$CMD_FOUND" ]; then
+		# Internal commands use underscores rather than hyphens
+		local converted_command_name=$(echo "$COMMAND" | tr '-' '_')
```

Comment:
Better (bc no process is forked)

    converted_command_name=${COMMAND//-/_}

Reply:
Sure, it was always like that, I only moved it closer to the call site. Feel free to push that change to the feature branch before I merge it.